plugins/shorewall/shorewall_log

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python3
```

Comment:
Please use `/usr/bin/env python3` in order to support virtualenvs.

Reply:
done